tests/umockdev.c

```diff
@@ -409,6 +409,8 @@ test_fixture_add_canon(UMockdevTestbedFixture * fixture)
 		"E: DRIVER=usb\n"
 		"E: BUSNUM=001\n"
 		"E: DEVNUM=001\n"
+		"E: MAJOR=189\n"
```

Comment:
Just FYI, this is the correct thing here and it is needed because the `is_initialized` check in the enumerator is special and considers any device without a device number always as initialized (`udev_device_get_is_initialized` behaves differently there).